docs/guide/snaps.md

```diff
@@ -56,21 +56,21 @@ If you're ready to jump in and see how a Snap looks on the backend, follow the s
 
 Open up a new terminal window, and do the following:
 
-- clone the [template snap](https://github.com/MetaMask/template-snap) repo
-- `cd` into the repo, and run `nvm use`
-  - this will install and/or use the recommended NVM version from `.nvmrc`
-- run `yarn install && yarn build:clean && yarn serve`
-  - this runs the commands needed to install dependencies
-  - if there is a `dist/` folder, it is cleaned before the build command is ran
-  - finally, we serve the dapp on `http://localhost:8080`
+- Clone the [template snap](https://github.com/MetaMask/template-snap) repo.
+- `cd` into the repo, and run `nvm use`.
+  - This will install and/or use the recommended NVM version from `.nvmrc`.
+- Run `yarn install && yarn build:clean && yarn serve`.
+  1. This first runs the commands needed to install dependencies.
+  1. If there is a `dist/` folder, it is then cleaned before the build command is run.
```

Comment:
Is it intentional to not number the list entries correctly here? 🤔

Reply:
Yes! This way you don't really need to think about what number something is — it'll get converted to the right number by Markdown when rendered. This is in the CommonMark spec: https://spec.commonmark.org/0.30/#start-number